Misc/NEWS.d/next/Security/2023-03-07-20-59-17.gh-issue-102153.14CLSZ.rst

```diff
@@ -0,0 +1,3 @@
+:func:`urllib.parse.urlsplit` now strips leading and trailing C0 control and
```

Comment:
Is the "trailing" correct here?

Reply:
good catch, i thought I had found all of those references. fixed.